velox/exec/SortBuffer.cpp

```diff
@@ -25,12 +25,14 @@ SortBuffer::SortBuffer(
     const std::vector<CompareFlags>& sortCompareFlags,
     velox::memory::MemoryPool* pool,
     tsan_atomic<bool>* nonReclaimableSection,
+    const PrefixSortConfig&& prefixSortConfig,
```

Comment:
why const &&?

A convention is to pass by value and std::move.

Reply:
oh ,  'pass by value' is better way since c++11.

https://en.cppreference.com/w/cpp/language/copy_elision